lib/matplotlib/backends/backend_pdf.py

```diff
@@ -550,7 +550,7 @@ def close(self):
                                for val in six.itervalues(self.alphaStates)]))
         self.writeHatches()
         self.writeGouraudTriangles()
-        xobjects = dict(six.itervalues(self.images))
+        xobjects = dict(x[1:] for x in six.itervalues(self.images))
```

Comment:
so, this would indicate to me that there is some sort of change to `self.images` for the Pdf backend object (or whatever class this is)? We probably should look closely to make sure all usages of self.images is changed appropriately. There might also be some API breakage considerations, although I am not sure why anybody would be subclassing the pdf backend...